tests/query/highlights/promql/regex.promql

```diff
@@ -0,0 +1,9 @@
+# vim: ft=promql
```

Comment:
Modelines go at the end of the file (separated by a blank line).

Reply:
done